src/server/pkg/collection/collection.go

```diff
@@ -244,7 +244,7 @@ func (c *readWriteCollection) DeleteAll() {
 		// Delete indexes
 		indexDir := c.prefix
 		indexDir = strings.TrimRight(indexDir, "/")
-		c.stm.DelAll(fmt.Sprintf("%s__index_%s/", indexDir, index))
+		c.stm.DelAll(fmt.Sprintf("%s__index_%v/", indexDir, index))
```

Comment:
What's the point of these changes?

Reply:
The new vetting as part of `go build` complained / err'd out at these lines. Basically it said something like 'mismatch types string vs indexCol` or something like that. One of the statements was missing the output value.

Reply:
Ahh, I see, so this is actually a bit of a pain. We've actually known about this bug for a while but haven't fixed it because doing so required migration. The issue here is that the result of these `fmt.Sprintf`s is changing, which means that we won't be able to find the indexes we computed in previous versions. It might just be time to bite the bullet and do this migration since we're doing other stuff.